test.go

```diff
@@ -0,0 +1,119 @@
+package main
```

Comment:
is this file used for testing only, can remove now?

Reply:
yep